sdk/storage/azblob/internal/exported/blob_batch.go

```diff
@@ -0,0 +1,271 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package exported
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/uuid"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/shared"
+	"io"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"net/textproto"
+	"strconv"
+	"strings"
+)
+
+type BlobBatchOperationType string
+
+const (
+	batchIdPrefix                                    = "batch_"
+	httpVersion                                      = "HTTP/1.1"
+	httpNewline                                      = "\r\n"
+	BatchDeleteOperationType  BlobBatchOperationType = "delete"
```

Comment:
Nit.  The convention is to separate logically grouped constants into their own `const` block.  And since these are strongly typed, the block should go under the respective type.

Reply:
Done